applications/vessel-history/src/features/profile/components/Info.tsx

```diff
@@ -159,20 +160,20 @@ const Info: React.FC<InfoProps> = (props): React.ReactElement => {
                 )}
               </div>
               <InfoField
-                vesselName={vessel.shipname ?? ''}
+                vesselName={vessel.shipname ?? DEFAULT_EMPTY_VALUE}
```

Comment:
I see that vesselName is not used in InfoField, can be deleted?

Reply:
No, because it's used to send to the InfoFieldHistory to set the title of the popup

Reply:
oh, I didn't see it, but I think is the same of the other comment so... we don't need to set every time we call the InfoField